libs/chain-signatures/contract/Cargo.toml

```diff
@@ -78,9 +78,10 @@ assert_matches = "1.5.0"
 rand = "0.8"
 tokio = { version = "1", features = ["full"] }
 sha2 = "0.10.8"
-reqwest = { version = "0.11", features = ["json"] }
-base64 = "0.21"
+reqwest = { version = "0.11", features = ["json"] } # TODO: Remove
+base64 = "0.21" # TODO: Remove
```

Comment:
by remove you mean to use the workspace version?

Reply:
Oh! Forgot about this. This dependency should be unused. Thanks for spotting.

Reply:
Done in https://github.com/near/mpc/pull/1031/commits/35fafcabce91932be9a433e0e18d686056a925a2

Reply:
Ahhh I'm blind :sweat_smile: Let's also forbid unused crate dependencies.

Reply:
Fixed in https://github.com/near/mpc/pull/1031/commits/06362c969e1995ea31e737e45e85348ed2b51868. Set to auto-merge again. I'll forbid unused dependencies in a follow-up since it's bound to reveal additional crates that needs to be pruned.